test/integration/auth.js

```diff
@@ -87,6 +85,51 @@ test.serial('duplicate name fails with badKnownName', async t => {
   t.is(resp.body.kind, 'badKnownName')
 })
 
+test.serial('succeeds with goodUserUpdate', async t => {
+  const user = await database.auth.getUserByEmail({
+    email: testUser.email
+  })
+
+  const nextUser = util.generateTestUser()
+
+  const authToken = await auth.token.getToken(auth.token.tokenKinds.auth, user.id)
+
+  const resp = await request(app)
+    .patch(process.env.API_ENDPOINT + '/users/me')
+    .set('Authorization', ' Bearer ' + authToken)
+    .send({
+      name: nextUser.name,
+      division: nextUser.division
+    })
+    .expect(responseList.goodUserUpdate.status)
+
+  t.is(resp.body.kind, 'goodUserUpdate')
+
+  const respUser = resp.body.data.user
+  t.is(respUser.name, nextUser.name)
+  t.is(respUser.email, testUser.email)
+  t.is(respUser.division, nextUser.division)
+
+  testUser.name = respUser.name
+  testUser.email = respUser.email
+  testUser.division = respUser.division
+})
```

Comment:
I don't think this code is guaranteed to run if the above tests fail. It might be a good idea to put them above the t.is checks to make sure goodUserDelete test doesn't fail due to this one failing.